### PR TITLE
dashboard,taffy: Don't create intermediate release for taffy deploys

### PR DESCRIPTION
**Pull request by jvatic**
```
Usage: taffy <app> <repo> <branch> <rev> [-e <var>=<val>]... [-m <key>=<val>]...
```

### CODE REVIEW

**Comment by jvatic** [comment]
This is ready for review

File: taffy/main.go [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/pkg/version"
+)
+
+func init() {
+	log.SetFlags(0)
+}
+
+const DEST = "/tmp/app"
```
Comment (titanous) [inline taffy/main.go]: We don't use all caps constants in Go, maybe `DestPath`?

File: taffy/main.go [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+			args = append(args, fmt.Sprintf("%s=%s", k, v))
+		}
+	}
+	cmd := exec.Command("/bin/flynn-receiver", args...)
+	cmd.Stdin = &buf
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	if err := gitArchive(rev, &buf); err != nil {
+		return err
+	}
```
Comment (titanous) [inline taffy/main.go]: This should be run in a goroutine with an `io.Pipe` instead of buffering the entire tarball in memory.

**Comment by jvatic** [comment]
@titanous comments addressed

**Comment by titanous** [comment]
LGTM

**Pull request merged**